ProcessLib/LIE/HydroMechanics/LocalAssembler/IntegrationPointDataFracture.h

```diff
@@ -50,6 +50,8 @@ struct IntegrationPointDataFracture final
     Eigen::MatrixXd C;
     double integration_weight;
 
+    Eigen::Vector3d darcy_velocity;
```

Comment:
Yes, it is not recalculated for the output (in the `computeSecondaryVariable...` call).

For the coffee discussion: It might be worth to add yet another function to the `Process` interface for updating the local states after a new solution was computed. This matters not only for output, as you mention, but maybe also for the staggered coupling.